bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties

```diff
@@ -271,6 +271,37 @@ channel-type.luxtronikheatpump.temperatureRoom.label = Raumtemp. Istwert
 channel-type.luxtronikheatpump.temperatureRoomTarget.label = Raumtemp. Sollwert
 channel-type.luxtronikheatpump.temperatureHotWaterTop.label = Temp. Brauchwasser Oben
 channel-type.luxtronikheatpump.frequencyCompressor.label = Verdichterfrequenz
+channel-type.luxtronikheatpump.channel232.label = Kanal 232
```

Comment:
Please update the default translation file : luxtronikheatpump.properties

I would recommend to not update the German translation in this PR but to do it later in Crowdin when the PR is merged.
If you do it in that PR, this will require a manual action in Crowdin to load your changes ; there is a risk to loose translations.

Reply:
Ah ok. I'm not very familiar with crowdin yet. We are using weblate in our projects, which has some kind of sync in both directions. But I can also add them to Crowdin once this PR was merged.

Reply:
But update the default translation file with English properties.

Reply:
You just have to run this command: `mvn i18n:generate-default-translations` at the root of your binding.

Reply:
ok. did that and pushed the update

Reply:
Just remove all changes in the German properties.
You (or someone) will translate in German later in Crowdin.